src/runtime/metatype.cs

```diff
@@ -22,6 +22,15 @@ public static IntPtr Initialize()
             return PyCLRMetaType;
         }
 
+        public static void Release()
+        {
+            if (Runtime.Refcount(PyCLRMetaType) > 1)
```

Comment:
Not sure I understand the reasoning behind `> 1` condition.
In fact, why even use `Refcount`? This class should have its own way to track if `PyCLRMetaType` is allocated.

Reply:
I think the reasoning is the same as with the similar pattern in typemanager.cs#38. The only way I can see this class tracking it's allocation is to look if the `PyCLRMetaType` is `IntPtr.Zero` or not.

Reply:
The logic is: If the refcount is 1, then at line 31 when decref, it'll go to zero and the entire type will be released. In that case we don't care about clearing the slots. It's a standard optimization in a ref-counting framework.

It's correct assuming that ReleaseTypeSlots doesn't also release refcounts that otherwise won't be released.

That said: Do we have coverage on line 29? I'm not sure I really understand under what circumstance it happens.